lua/astronvim/utils/ui.lua

```diff
@@ -208,4 +208,16 @@ function M.toggle_url_match()
   require("astronvim.utils").set_url_match()
 end
 
+--- Toggle foldcolumn=0|1|2
+function M.toggle_foldcolumn()
+  if vim.wo.foldcolumn == "0" then
+    vim.wo.foldcolumn = "1"
+  elseif vim.wo.foldcolumn == "1" then
+    vim.wo.foldcolumn = "2"
+  else
+    vim.wo.foldcolumn = "0"
+  end
```

Comment:
Should this be between 0/1/2? It probably makes more sense to make the toggle just between 1 and 0. Fold column can be anything between 0 and 9. We can just assume anything other than 1 is user set which is totally fine.